pylj/forcefields.py

```diff
@@ -100,6 +100,11 @@ def buckingham(dr, constants, force=False):
     float: array_like
         The potential energy or force between the particles.
     """
+    if not isinstance(dr, np.ndarray):
+        if isinstance(dr, list):
+            dr = np.array(dr, dtype='float')
+        elif isinstance(dr, float):
+            dr = np.array([dr], dtype='float')
```

Comment:
I think it might be the case that this isn't necessary if you change line 111 and 114 so that 
``` 
-constants[1] * dr
``` 

is changed to 
```
np.multiply(-constants[1], dr)
```

Could you possibly test this out?

Reply:
You were correct, np.multiply handles that. Hope viva prep is going well!

Reply:
Awesome cheers for the pull request! Viva prep well enough that I passed!

Reply:
Congrats!